posix.mak

```diff
@@ -79,7 +79,9 @@ MOD_EXCLUDES_PRERELEASE=$(addprefix --ex=, gc. rt. core.internal. core.stdc.conf
 	std.algorithm.internal std.c. std.concurrencybase std.internal. std.regex.internal.  \
 	std.windows.iunknown std.windows.registry etc.linux.memoryerror	\
 	std.experimental.ndslice.internal std.stdiobase \
-	tk. msvc_dmc msvc_lib)
+	tk. msvc_dmc msvc_lib \
+	ddmd.libmach ddmd.libmscoff ddmd.objc_glue \
+	ddmd.scanmach ddmd.scanmscoff)
```

Comment:
> I had a quick look at whether it was possible to enable them. At least in the case of the OS X stuff, it was importing stuff from Druntime, and those modules had a `version(OSX):` on top.

Yeah, I expected as much.

> I'm not sure if that makes sense for things such as type declarations; I have a Druntime PR open to make the Windows headers usable on other platforms (for things such as .bmp and .exe/.dll file formats), but that grew the size of the built library file by a disproportionate amount due to all the .init / TypeInfo clutter...

Disregarding the bloat issue for a moment, I certainly find this valuable. (I also wrote my own bmp file reader/writer, though in my case writing the definitions by hand had high educational value :D) Also, I would love to be able to check dlang.org if a certain definition is present in `core.sys.windows`. OTOH, `core.sys.windows` being probably the largest D subpackage should better be excluded from the sidebar.

I faintly remember reading about the .init / TypeInfo code bloat issue on your PR. Is there a bug report on that? I would certainly expect those modules to behave like header-only libraries which don't get compiled.